app/containers/Menu.js

```diff
@@ -0,0 +1,649 @@
+          name: 'Cape Cod',
+          ingredients: '1 part vodka, 2 part cranberry juice',
+          rating: 4.39,
+          url: 'http://www.barmeister.com/drinks/recipe/297/?o=top100'
+        },
+        {
+          name: 'Red Devil',
+          ingredients: '1 oz Sloe gin, 1/2 oz Southern Comfort, 1/2 oz Amaretto, 1/2 oz Vodka, 1/2 oz lime juice, 1/2 oz Triple Sec, 5 oz Orange Juice',
+          rating: 4.38,
+          url: 'http://www.barmeister.com/drinks/recipe/1251/?o=top100'
+        },
+        {
+          name: 'Gladiator',
+          ingredients: '1/2 oz Southern Comfort, 1/2 oz Amaretto, 1 part 7Up, 1 part Orange Juice',
+          rating: 4.38,
+          url: 'http://www.barmeister.com/drinks/recipe/672/?o=top100'
+        },
+        {
+          name: 'Lemon Drop',
+          ingredients: '1 oz vodka, a pinch of sugar, a slice of lemon',
+          rating: 4.35,
+          url: 'http://www.barmeister.com/drinks/recipe/898/?o=top100'
+        },
+        {
+          name: 'Juicy Pussy',
+          ingredients: '3/4 oz Peach Schnapps, 1/4 oz Bailey\'s Irish Cream, 1/2 glass of orange juice',
+          rating: 4.36,
+          url: 'http://www.barmeister.com/drinks/recipe/853/?o=top100'
+        },
+        {
+          name: 'Singapore Sling',
+          ingredients: '1 1/2 oz Gin, 1/2 oz Brandy, 3 dashes benedictine, dash of grenadine, dash of lemon juice, fill with club soda',
+          rating: 4.37,
+          url: 'http://www.barmeister.com/drinks/recipe/1393/?o=top100'
+        }
+      ],
+    });
+    return {};
+  },
+
+  drinkListReturn: function () {
+    return this.state.drinkList.map(function (name, index) {
+      //console.log({name});
+      //Need URL with <a></a>
+      return <li><a href={name.url} class="list-group-item active"><h4 key={index}>{name.name}</h4><p>{name.ingredients}</p></a></li>;
+    });
+  },
+
+  popularSort: function () {
+    this.state.drinkList.sort(function (drink1, drink2) {
+      return drink2.rating - drink1.rating;
+    });
+    /*this.setState({
+
+    });*/
+  },
+
+  alphabeticalSort: function () {
+    var compared
+    this.state.drinkList.sort(function (index1, index2) {
+        var drink1 = index1.name;
+        var drink2 = index2.name;
+        if (drink1 < drink2) {
+          return -1;
+        }
+
+        if (drink1 > drink2) {
+          return 1;
+        }
+
+      // names must be equal
+      return 0;
+      });
+
+      /*this.setState({
+
+      });*/
+
+  },
+
+  shouldComponentUpdate: function () {
+
+    if (this.lastSort === this.state.sort) {
+      return false;
+    }
+    this.lastSort = this.state.sort;
+    return true;
+  },
+
+  setSortingPopularFn: function() {
+    if(this.state.sort === "Alphabetical") {
+      this.setState({
+        sort: "Popularity",
+        alphabeticalButtonDisabled: false,
+        popularButtonDisabled: true,
+      });
+      this.popularSort();
+    }
+  },
+
+  setSortingAlphabeticalFn: function () {
```

Comment:
minor: probably best to combine these methods to something like sortDrinks(category, order) where category would be Alphabetical or Popularity and order would be descending or ascending. Even better, you could put category and order in enums like
```
DrinkList.SortingProps = {
  SortAlphabetical: "Alphabetical",
  SortPopular: "Popular", 
 ...
}
```

then you can use that enum instead of "magic strings and numbers" and call sort like `sort(DrinkList.SortingProps.SortAlphabetic, DrinkList.SortingProps.Ascending)`  

This helps you get away from typing strings into your code in multiple places and therefor reduces chance of human error (types)